src/player-service/playerRoutes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { getPlayerById, getPlayersByTeam, placePlayerOnTransferList, transferPlayer } from './playerController';
+import { authMiddleware } from '../libs/common/middleware/auth';
+
+const router = Router();
+
+router.use(authMiddleware);
+
+router.get('/:teamId', getPlayersByTeam);
```

Comment:
Why not have a list API that we can pass teamId instead?

Reply:
makes sense